data/templates/cloud-config.yaml

```diff
@@ -0,0 +1,35 @@
+#cloud-config
+write_files:
+  - path: /etc/rc.local
+    permissions: "0755"
+    owner: root
+    content: |
+      #!/bin/bash
+      modprobe ipmi_devintf
+      wget -O /tmp/micro.tar.xz <%= api.server %>/micro.tar.xz
```

Comment:
does it mean microkenrel? if yes, how about 'microkernel', if not, maybe it could be micore<what>